playground/nats/Nats.AppHosting/Program.cs

```diff
@@ -0,0 +1,10 @@
+using Aspire.Hosting.Nats;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var nats = builder.AddNats("nats", enableJetStream: true);
```

Comment:
You may wish to add a `PublishAsContainer(...)` extension method. Take a look what we have done for resources like Redis where we don't have `AddRedisContainer(...)` anymore but we've added a `PublishAsContainer()` method which can be used to tell the resource to write to the manifest as a `container.v0` resource.